### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by Ney2**
This is a PR for adding and reducers and actions for our book store application.
1. Install redux dependency.
2. Create a book file and add an action and reducer.
3. Create a categories file and add an action and reducer.
4. Create a configureStore file to combine the reducers.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// ACTION
+export const addBook = (book) => ({
+  type: 'ADD_BOOK',
+  data: book,
+});
+
+export const removeBook = (id) => ({
+  type: 'REMOVE_BOOK',
+  data: id,
+});
```
Comment (tresorsawasawa) [inline src/redux/books/books.js]: - [ ] As required for the project, kindly make sure your `ADD_BOOK` and `REMOVE_BOOK` are defined as variables and the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

File: src/redux/catagories/catagories.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// ACTION
+export const checkStatus = () => ({
+  type: 'CHECK_STATUS',
+});
```
Comment (tresorsawasawa) [inline src/redux/catagories/catagories.js]: - [ ] As I mentioned in the previous comment, kindly make sure the `CHECK_STATUS` action is defined as a variable and then its value follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Ney2 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @Ney2 , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**